### DISCUSSION TITLE
Ray tracing and get received power data

### DISCUSSION THREAD

**Discussion by JoieMing** [NVlabs/sionna#215] [Q&A]
I have collected SSB_RP data for a coverage area in the real world. My target is to use RT provided by Sionna to obtain the simulation data and compare it with the SSB_RP data. 

How can I obtain or save the average attenuation data rather than the figure provided by the coverage map?  How to compute the received power for receivers located in different small cells of the entire region? As for the sionna.rt.Transmitter, I didn't find a parameter related to transmit power. Cloud you please kindly let me know how to assign different transmit power for different transmitters?

Thank you in advance for your time and help.

**Comment by jhoydis** [comment]
Hi,

You can get access to the data by calling the method [``CoverageMap.as_tensor()``](https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.CoverageMap.as_tensor).

Transmitters do not have a transmit power parameter as it is always assumed to be equal to one.

You can compute the received power for each transmitter by simply multiplying its transmit power with the channel gain as predicted by the coverage map.

**Reply by JoieMing** [reply]
the output of CoverageMap.as_tensor() is [num_tx, num_cells_y, num_cells_x], what is the mapping relationship between the cells_x and the x coordinate?

**Reply by jhoydis** [reply]
Hi,

I think that there is an undocumented property of the coverage map called ``CoverageMap.cell_centers``.
You can checkout the [source code](https://nvlabs.github.io/sionna/_modules/sionna/rt/coverage_map.html#CoverageMap).
